docs/databases/postgres.md

```diff
@@ -4,4 +4,118 @@ title: Using PostgreSQL with Graphback
 sidebar_label: PostgreSQL
 ---
 
-## TODO
\ No newline at end of file
+[PostgreSQL](https://www.postgresql.org/) is a powerful, open source object-relational database system with over 30 years of active development that has earned it a strong reputation for reliability, feature robustness, and performance. [Graphback Knex Provider](https://www.npmjs.com/package/@graphback/runtime-knex) package provides instant integration to your PostgreSQL database, giving you a full implementation of the [CRUD API](../crud/introduction.md). 
+
+Additionally, Graphback comes with the [GraphQL Migrations](../graphql-migrations/intro.md) package that can be used to perform migrations against your database using the business models defined in the GraphQL Schema. 
+
+The provider is built on top of [Knex.js](http://knexjs.org/), a flexible SQL query builder.
+
+# Installation
+
+Install with npm:
+
+```bash
+npm install @graphback/runtime-knex knex
```

Comment:
https://github.com/aerogear/graphback/blob/master/packages/graphback-runtime-knex/package.json#L41

Knex is there, but I know @craicoverflow asked to remove it.